pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -238,7 +245,7 @@ func (b *volumeBinder) FindPodVolumes(pod *v1.Pod, node *v1.Node) (unboundVolume
 			if selectedNode, ok := claim.Annotations[pvutil.AnnSelectedNode]; ok {
 				if selectedNode != node.Name {
 					// Fast path, skip unmatched node
-					return false, boundVolumesSatisfied, nil
+					return []string{ErrReasonBindConflict}, nil
```

Comment:
boundVolumesSatisfied could also be false at this point.

Reply:
I know (https://github.com/kubernetes/kubernetes/pull/88160#discussion_r379496960). I decided to keep it simple here because in the end all that matters is that *some* reason is returned. I can add an if check for `boundVolumesSatisfied` here if you think it is important.

Reply:
I think it'll be nice to return all reasons if we know it instead of having it depend on ordering of checks

Reply:
If we allow the use of `goto` here, then it can be done without code duplication. Changed.